pkgs/by-name/sc/scopehal-apps/package.nix

```diff
@@ -78,10 +84,31 @@ stdenv.mkDerivation {
     moltenvk
   ];
 
-  cmakeFlags = lib.optionals stdenv.hostPlatform.isDarwin [
-    "-DCMAKE_INSTALL_RPATH=${lib.strings.makeLibraryPath [ vulkan-loader ]}"
+  cmakeFlags = [
+    "-DNGSCOPECLIENT_VERSION=${version}"
+  ];
+
+  patches = [
+    ./remove-git-derived-version.patch
```

Comment:
This block will reset `NGSCOPECLIENT_VERSION` to `unknown` even if we already supply a value for it.